resources/windows/app-updater-config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "notifyOnly": true
```

Comment:
Is this first version just going to notify the user that there's updates, or are we going to supply a different config later on?

Also, where is it going to check for these updates?

Reply:
This static config file is only bundled into zip version of Windows app because zip version can't use auto-updater. It should be read-only and change a button in the dialog as esethna had reviewed. Then the button will open https://mattermost.com/download/ .

![notifyonly-dialog](https://user-images.githubusercontent.com/14333569/46756285-52b2d080-cc84-11e8-82bd-99edaa9cbe0a.png)

The updater will check `latest.yml` and `latest-mac.json` under a base url. The base url is defined in `electron-builder.json`.

https://github.com/mattermost/desktop/blob/849800a80fb4c44242c1ddab824ccf9ba451890c/electron-builder.json#L4

The base url should be `https://releases.mattermost.com/desktop/` finally.

Reply:
Ah, I knew it was somewhere. So it'll look at `http://desktopupdate.mattermost.com/latest.yml` and `http://desktopupdate.mattermost.com/latest-mac.json` to get those updates. Could we switch those to use https?

Also, this doesn't really matter, but is there a reason you chose to mix yml and json files for those two?

Reply:
Sure, We can use https.

Mixing yml and json is derived from `electron-builder` spec. It generates them at build-time, so we can't control.

Reply:
Makes sense that that's why we switch between the two. It's a bit odd, but it's not a problem.

Using https is important though since it would ensure that the file is actually coming from us and the request isn't being intercepted by someone else